File: src/main/java/io/github/dqualizer/dqlang/types_new/adapter/constants/loadprofile/LoadIncrease.java
package io.github.dqualizer.dqlang.types_new.adapter.constants.loadprofile;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.ToString;

@Getter
@ToString
public class LoadIncrease {

    @JsonProperty("start_target")
    private int startTarget;
    @JsonProperty("end_target")
    private int endTarget;
    @JsonProperty("stages")
    private int stages;
    @JsonProperty("test_duration")
    private String testDuration;
    private int linear;
    private int quadratic;
    private int cubic;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/k6adapter/dqlang/constants/loadprofile/LoadIncrease.kt
package io.github.dqualizer.dqlang.archive.k6adapter.dqlang.constants.loadprofile

import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Getter
import lombok.ToString

@Getter
@ToString
class LoadIncrease {
    @JsonProperty("start_target")
    val startTarget = 0

    @JsonProperty("end_target")
    val endTarget = 0

    @JsonProperty("stages")
    val stages = 0

    @JsonProperty("test_duration")
    val testDuration: String? = null
    val linear = 0
    val quadratic = 0
    val cubic = 0
}
